src/components/Cardgrid.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {
+  CircularProgress,
+  FontIcon,
+  IconButton,
+} from 'material-ui';
+import {
+  GridList,
+  GridTile
+} from 'material-ui/GridList';
+import { muiThemeable } from 'material-ui/styles';
+
+const styles = {
+  image: {
+    width: 'initial',
+    height: 'initial',
+    maxWidth: '100%',
+    maxHeight: '100%',
+  },
+  load: {
+    width: '100%',
+    height: 135,
+    paddingTop: 40,
+    boxSizing: 'border-box',
+    backgroundColor: '#606060',
+    textAlign: 'center',
+  },
+  loadmore: {
+    color: 'white',
+    fontSize: 30,
+    cursor: 'pointer',
+    textDecoration: 'underline',
+    backgroundColor: 'transparent',
+    border: 'none',
+    outline: 'none',
+  }
+};
```

Comment:
let's discuss how we want to keep the styles. Can you please suggest a few best practices, with pros and cons?